### PR TITLE
Widget-Inventory-Planner-13 Write a hello world type page to introduce the WIP

### PR DESCRIPTION
**Pull request by appdev734**
Widget-Inventory-Planner-13 Write a hello world type page to introduce the WIP

### CODE REVIEW

**Comment by srumbum** [comment]
@appdev734 Hey , I looked at, looks OK

File: Hello_WIP.html [hunk 1]
```diff
@@ -27,4 +27,16 @@
...
+now using github,
+1) Cloned WIP to local
+2) Created branch: Widget-Inventory-Planner-13
+3) Checked out master
+4) Checked out Widget-Inventory-Planner-13
+5) making these changes
+
+-- would like to see branch committed to remote master
+ -- as storage, but not merged
+ -- see if can do on pull request
```
Comment (srumbum) [inline Hello_WIP.html]: So did the comment get attached to the file,  or is ti like another comment in conversation?

File: Hello_WIP.html [hunk 2]
```diff
@@ -27,4 +27,16 @@
 
 apd_dev_branch_02 - get this up to github via create pull request
```
Comment (appdev734) [inline Hello_WIP.html]: Would like to see a line number if is comment for code

File: Hello_WIP.html [hunk 3]
```diff
@@ -27,4 +27,16 @@
...
 previous one was just a commit/push without pull request
+
+
+now using github,
+1) Cloned WIP to local
+2) Created branch: Widget-Inventory-Planner-13
```
Comment (appdev734) [inline Hello_WIP.html]: OK, now it seems to be working for line number. Before we started at the bottom, so did not appear as such

Comment (appdev734) [inline Hello_WIP.html]: +now using github,
 +1) Cloned WIP to local
 +2) Created branch: Widget-Inventory-Planner-13

Go back to this and redo, until the local/remote github flow is well established in phpstorm an atom

File: Hello_WIP.html (hunk 1 above)
Comment (appdev734) [inline Hello_WIP.html]: See what it means to start a review

Comment (appdev734) [inline Hello_WIP.html]: So the review tells everyone that what? we have moved from discussion and have now entered review?

**Comment by appdev734** [comment]
@srumbum So, this is just another general comment, not a code comment, not a review comment, just a comment comment.

**Comment by appdev734** [comment]
What happens when you have a local repository and you
1) Shelve changes
2) Update from remote master
3) remote master has either deleted the file you had shelved or made other changes to it
4) Does the shelve show as comparison, replace the file on the local, or what?

**Pull request merged**